cmd/main.go

```diff
@@ -962,11 +962,11 @@ func runOpenshiftReleaseSyncerIfNeeded(
 		if err != nil {
 			failOnError(err, "error occurred while creating release sources handler")
 		}
-		log.WithField("pkg", "releasesources").Info("Running Openshift Releases Syncer at startup")
-		err = releaseSourcesHandler.SyncReleaseImages()
-		if err != nil {
-			failOnError(err, "error occurred while syncing release images at startup")
-		}
+
+		failOnError(lead.RunWithLeader(context.Background(), func() error {
```

Comment:
`RunWithLeader` is a concurrency control. This means that every replica will run this code.

This is fine for automigration because no work is done once the database is fully migrated, but for this it will lead to us removing and re-adding all releases for every replica, right? Are we okay with that behavior?

Reply:
Yes you are right, I tested it and it doesn't have performance impact as it will happen only once per instance. I agree this is not ideal but I don't see any other option for running this only by one instance, as the table need to be cleaned up anyway if this is not the there are records in the DB already

Reply:
I can try to explore a more ideal solution after